clickhouse-native-jdbc/src/main/java/com/github/housepower/jdbc/wrapper/SQLDatabaseMetadata.java

```diff
@@ -14,22 +14,23 @@
 
 package com.github.housepower.jdbc.wrapper;
 
+import com.github.housepower.jdbc.exception.NotImplementedException;
 import com.github.housepower.jdbc.log.Logging;
 
 import java.sql.*;
 
-public interface SQLDatabaseMetadata extends DatabaseMetaData, Logging {
+public interface SQLDatabaseMetadata extends DatabaseMetaData, SQLWrapper, Logging {
 
     @Override
     default int getDriverMajorVersion() {
         logger().debug("invoke unimplemented method #getDriverMajorVersion()");
-        return 0;
+        throw new NotImplementedException("unimplemented method #getDriverMajorVersion()");
     }
 
     @Override
     default int getDriverMinorVersion() {
         logger().debug("invoke unimplemented method #getDriverMinorVersion()");
-        return 0;
+        throw new NotImplementedException("unimplemented method #getDriverMinorVersion()");
```

Comment:
Why throw this?  Maybe currently return 0 is ok ?

Reply:
Move all implement out the wrapper interface, it's implemented in `ClickHouseDatabaseMetadata`